stdlib/_typeshed/__init__.pyi

```diff
@@ -115,6 +118,7 @@ class SupportsItemAccess(SupportsGetItem[_KT_contra, _VT], Protocol[_KT_contra,
 # These aliases are simple strings in Python 2.
```

Comment:
We should probably remove this comment, as it will be outdated soon and only applies to some of the aliases now.

Reply:
```suggestion
```